### PR TITLE
CB-11009. Add disk and memory data to cdp doctor report.

### PR DESCRIPTION
**Pull request by oleewere**
See detailed description in the commit message.

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt-common/salt/filecollector/template/cdp-doctor-commands.yaml.j2 [hunk 1]
```diff
@@ -6,6 +6,14 @@ commands:
       report: false
     - command: netstat -tlpn
       output: /tmp/doctor_netstat.txt
+    - command: free -h
+      output: /tmp/doctor_free.txt
+    - command: ps -eo pid,ppid,cmd,%mem,%cpu --sort=-%mem | head
+      output: /tmp/doctor_top_proc.txt
+    - command: df -h
+      output: /tmp/doctor_disk_full.txt
```
Comment (pdarvasi) [inline orchestrator-salt/src/main/resources/salt-common/salt/filecollector/template/cdp-doctor-commands.yaml.j2:14]: why is the name dist_full.txt? df is short for disk free

Comment (oleewere) [reply orchestrator-salt/src/main/resources/salt-common/salt/filecollector/template/cdp-doctor-commands.yaml.j2:14]: it's just accidental...so i have created one check for only the **logs** and one for the **full** filesystem based on the command name. can be changed if you think disk free as name is better (to be alias of the command)

Comment (oleewere) [reply orchestrator-salt/src/main/resources/salt-common/salt/filecollector/template/cdp-doctor-commands.yaml.j2]: whatever, fixed it to not confuse anyone by the naming

**Review by attilapalfi92 — APPROVED** [review]

**Comment by biharitomi** [comment]
@oleewere to check volume related problems could you please add the `lsblk` command output to collected results?

**Comment by oleewere** [comment]
@biharitomi ..if you write it 1 minute earlier, almost:) ... i can create a follow up PR

**Comment by biharitomi** [comment]
Thanks @oleewere , that will be nice to have that way too.

**Pull request merged**